product_pricelist_direct_print/wizards/product_pricelist_print.py

```diff
@@ -95,6 +96,20 @@ def _compute_product_price(self):
         else:
             self.product_price = price
 
+    def get_price_for_pricelist(self, pricelist, product):
+        # TODO enable this instead
```

Comment:
Is it wip ?

Reply:
More like an open question for reviewers.
Is there a benefit for using a computed field depending on the context rather than using a method?

Reply:
I think your right. I'm not sure to understand the current design. 
however, changing the code could break possible module overloading this module. Not sure what to think here.
In all cases, if you want to introduce questions, please add better a note in the ROADMAP.rst file that mention "replace computed function xxx by yyy when porting this module". i think it's better than adding commented code with TODO.
What do you think ?

Reply:
I agree with @legalsylvain, adding these changes to the module at this stage could cause inheritance issues in other modules that rely on them.